filters/CMakeLists.txt

```diff
@@ -86,6 +86,7 @@ set(incs
   "include/pcl/${SUBSYS_NAME}/morphological_filter.h"
   "include/pcl/${SUBSYS_NAME}/local_maximum.h"
   "include/pcl/${SUBSYS_NAME}/model_outlier_removal.h"
+  "include/pcl/${SUBSYS_NAME}/experimental/functor_filter.h"
```

Comment:
Already present in line 61. I think you have to add an `PCL_ADD_INCLUDES` at the bottom. The pattern with the "experimental" folder is not common in PCL, though. Maybe the filter is already mature enough to just put `functor_filter.h` next to the other `*.h` files?

Reply:
Oh yeah. We wanted to prevent maturing the API before experimenting with it. Look at the PRs for executor by @shrijitsingh99